rows_test.go

```diff
@@ -260,6 +308,296 @@ func TestQuerySingleRow(t *testing.T) {
 	}
 }
 
+func TestQueryRowBytesInvalidatedByNext(t *testing.T) {
+	invalid := []byte(`☠☠☠ MEMORY OVERWRITTEN ☠☠☠ `)
+	type want struct {
+		Initial  []byte
+		Replaced []byte
+	}
+	testTable := map[string]struct {
```

Comment:
I'm using table driven tests here, which are not currently being used by other tests. Given I'm trying to test a series of different cases, with the same basic structure, table-driven made sense, but I can rewrite these without it if needed for consistency

Reply:
Looks like test tables can't be used, since build compatibility is checked all the way back to Go 1.2:

```
# github.com/DATA-DOG/go-sqlmock
./rows_test.go:431: t.Run undefined (type *testing.T has no field or method Run)
./rows_test.go:560: t.Run undefined (type *testing.T has no field or method Run)
FAIL	github.com/DATA-DOG/go-sqlmock [build failed]
```